cypress/e2e/specialFeatures/liteSiteWeight/index.cy.js

```diff
@@ -0,0 +1,97 @@
+import runTestsForPage from '#nextjs/cypress/support/helpers/runTestsForPage';
+
+import assertPageWeight from './assertions/liteSiteWeight';
+
+const tests = [assertPageWeight];
+const liteTestSuites = [
+  {
+    path: '/hindi.lite?renderer_env=live',
+    service: 'hindi',
+    runforEnv: 'local',
+    pageType: 'Home',
+    tests,
+  },
+  {
+    path: '/mundo/articles/cddylv9g8z0o.lite?renderer_env=live',
+    runforEnv: 'local',
+    pageType: 'Optimo Article',
+    tests,
+  },
+  {
+    path: '/nepali/bbc_nepali_radio/liveradio.lite?renderer_env=live',
+    runforEnv: 'local',
+    pageType: 'Live Radio',
+    tests,
+  },
+  {
+    path: '/arabic/media-53135426.lite?renderer_env=live',
+    runforEnv: 'local',
+    pageType: 'CPS Media Article with Live Stream',
+    tests,
+  },
+  {
+    path: '/marathi/popular/read.lite?renderer_env=live',
+    runforEnv: 'local',
+    pageType: 'Most Read',
+    tests,
+  },
+  {
+    path: '/gahuza/bbc_gahuza_radio/programmes/p0340x2m.lite?renderer_env=live',
+    runforEnv: 'local',
+    pageType: 'On Demand Audio - Brand',
+    tests,
+  },
+  {
+    path: '/gahuza/bbc_gahuza_radio/w3ct1v5v.lite?renderer_env=live',
+    runforEnv: 'local',
+    pageType: 'On Demand Audio - Episode',
+    tests,
+  },
+  {
+    path: '/gahuza/podcasts/p07yh8hb.lite?renderer_env=live',
+    runforEnv: 'local',
+    pageType: 'Podcast - Brand',
+    tests,
+  },
+  {
+    path: '/gahuza/podcasts/p07yh8hb/p094vs2n.lite?renderer_env=live',
+    runforEnv: 'local',
+    pageType: 'Podcast - Episode',
+    tests,
+  },
```

Comment:
I've noticed that there's a large delta between local + live for many of these URLs (if we look at the output of the E2E tests) - I believe it's because the radio schedule is enabled on live, but not on local (which uses test toggles). We should probably ensure that there's parity between local + live, so that the results are more accurate